appveyor.yml

```diff
@@ -1,3 +1,19 @@
+# The config expects the following environment variables to be set:
+#  - "GN_CONFIG" Build type. One of {'debug', 'testing', 'release'}.
+#  - "GN_EXTRA_ARGS" Additional gn arguments for a build config,
+#      e.g. 'fatal_linker_warnings=false enable_linux_installer=false'.
+#      Use it to set 'target_cpu' for example.
+#  - "ELECTRON_RELEASE" Define it to upload binaries on success.
+#  - "NPM_CONFIG_ARCH" E.g. 'ia32'. Is used to build native Node.js modules.
```

Comment:
This could also be windows specific, eg `x86`.

Reply:
Done